contracts/authorizers/RequesterAuthorizer.sol

```diff
@@ -5,14 +5,11 @@ import "@openzeppelin/contracts/metatx/ERC2771Context.sol";
 import "./interfaces/IRequesterAuthorizer.sol";
 
 /// @title Abstract contract to be inherited by Authorizer contracts that
-/// temporarily or permanently authorizes requesters for Airnode–endpoint pairs
-/// or Airnodes
-/// @dev An authorization for an Airnode with endpoint ID `bytes32(0)`
-/// represents a blanket authorization across all endpoints of the Airnode
+/// temporarily or permanently authorizes requesters for Airnodes
 abstract contract RequesterAuthorizer is ERC2771Context, IRequesterAuthorizer {
     struct AuthorizationStatus {
-        uint64 expirationTimestamp;
-        uint192 indefiniteAuthorizationCount;
+        uint32 expirationTimestamp;
```

Comment:
Using 32 bit timestamps for consistency

Reply:
Should we also consider doing the same for `expirationTimestamp` in Allocator contracts? https://github.com/api3dao/airnode-protocol-v1/blob/remove-endpoint-authorization/contracts/allocators/interfaces/IAllocator.sol#L43

Reply:
Will do